app/assets/stylesheets/pages/course_outline_page.scss

```diff
@@ -41,3 +41,20 @@
   margin-top: $xl;
   margin-bottom: $xl;
 }
+
+.radial-gradient-mask {
```

Comment:
might want to put this in a separate scss file since we're going to use this elsewhere.